apps/web/src/app/globals.css

```
@@ -88,3 +88,41 @@
   gap: 12px;
 }
+
+.evolution-rainbow-gradient-text {
+  /* Level 1 */
+
+  /* gradient-rainbow */
+  background: linear-gradient(132.51deg, #fffdc9 2.19%, #96e6d8 49.24%, #7dabf1 98.21%);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-clip: text;
+  text-fill-color: transparent;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)